pkg/apis/core/resource.go

```diff
@@ -34,6 +34,11 @@ func (rl *ResourceList) Memory() *resource.Quantity {
 	return rl.Name(ResourceMemory, resource.BinarySI)
 }
 
+// Swap returns the Swap limit if specified.
+func (rl *ResourceList) Swap() *resource.Quantity {
+	return rl.Name(ResourceSwap, resource.BinarySI)
+}
+
```

Comment:
Same comment as above, I don't think we need to define this since swap is not part of the pod spec.